src/harness/simple_crl_server.py

```diff
@@ -0,0 +1,105 @@
+#    Copyright 2018 SAS Project Authors. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+"""
+    A implementation of Certificate Revocation list Server.
+"""
+import logging
+import os
+import threading
+import urlparse
+from BaseHTTPServer import HTTPServer, BaseHTTPRequestHandler
+
+
+class CrlServerTestHarness(threading.Thread):
+  """
+  Implements a CRL server by creating a HTTP server and
+  serving CRL files The  CRL server runs as a separate thread
+  """
+
+  def __init__(self, host_name, port, crl_url, crl_file_name):
+    """
+    Constructor for  CRL test harness Requests from SAS Under Test
```

Comment:
Please search this file for "  " (double spaces) and remove the redundant ones.

Reply:
This   comment has been addressed now.